src/MyPlot/task/ClearPlotTask.php

```diff
@@ -90,6 +73,23 @@ public function onRun(int $currentTick) : void {
 			$this->pos->z = $this->plotBeginPos->z;
 			$this->pos->x++;
 		}
+		$bb = $this->plugin->getPlotBB($this->plot);
+		foreach($this->plugin->getPlotChunks($this->plot) as $chunk) {
+			foreach($chunk->getEntities() as $entity) {
```

Comment:
What i actually meant is foreach(level->getCollidedEntities(AABB), but this works too.